content/blog/2020/07/2020-07-27-machine-learning-plugin-coding-phase2.adoc

```diff
@@ -0,0 +1,150 @@
+---
+layout: post
+title: "Machine Learning Plugin project - Coding Phase 2 blog post"
+tags:
+- machinelearning
+- datascience
+- plugins
+- communitybonding
+- gsoc
+- gsoc2020
+author: loghijiaha
+description: "Coding Phase 2 of Google Summer of Code  2020"
```

Comment:
```suggestion
description: "Coding Phase 2 of Google Summer of Code 2020"
```

Would be great to add a short summary here (could be used as a sample for promotion in social media)